__tests__/app.test.js

```diff
@@ -63,6 +63,89 @@ describe('/api/articles/:article_id', () => {
                 });
         });
     });
+    describe("PATCH", () => {
+        test("200: updates votes and responds with updated object", () => {
+            const votes = {
+                inc_votes: 2,
+            };
+            const article_id = 1;
+            return request(app)
+                .patch(`/api/articles/${article_id}`)
+                .send(votes)
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.article).toMatchObject({
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.any(String),
+                        votes: 102
+                    });
+                    return db.query("SELECT * FROM articles WHERE article_id=$1", [
+                        article_id,
+                    ]);
+                })
+                .then(({ rows }) => {
+                    expect(rows[0]).toMatchObject({
+                        article_id: 1,
+                        title: "Living in the shadow of a great man",
+                        topic: "mitch",
+                        author: "butter_bridge",
+                        body: "I find this existence challenging",
+                        created_at: expect.anything(),
+                        votes: 102
+                    });
+                });
+        });
+        test('400: Invalid input', () => {
+            return request(app)
+                .patch('/api/articles/1')
+                .send({})
+                .expect(400)
+                .then(({ body }) => {
+                    expect(body).toEqual({ msg: "Invalid input" });
+                    return db.query("SELECT * FROM articles WHERE article_id=1");
```

Comment:
I think either use the article_id variable or hardcode in 1 in these database checks in your test (you've done it differently on line 86). I would argue just to hardcode it like you've done here.